examples/eth_provider/config/config.exs

```diff
@@ -0,0 +1,9 @@
+import Config
+
+config :wrapeth, EthProvider,
```

Comment:
This is weird. You have an application called `:eth_provider`, but you're configuring `:wrapeth`? Why?

Reply:
You're absolutely right! I'll change it to have the same name as the application. I'll make sure to pay more attention to this in the future.